src/snackbar/_snackbar.scss

```diff
@@ -0,0 +1,78 @@
+/**
+ * Copyright 2015 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@import "../variables";
+
+.mdl-snackbar {
+  position: absolute;
+  bottom: 0;
+  left: 50%;
+  margin-right: -50%;
+  cursor: default;
+  // Hard coded since the color is not present in any pallet.
+  background-color: #323232;
+  z-index: 10000;
+  max-height: 0;
+  border-radius: 2px;
+  display: table;
+  font-family: $preferred_font;
```

Comment:
I did this specifically to use the preferred fonts over performance (so Roboto.) Since Snackbars will typically be called after the font is ready it is just a nice enhancement.

I have no problem with removing the font set though and leaving it up to developers.